### PR TITLE
Add update Drupal core for security issue documentation

### PR DESCRIPTION
**Pull request by smirfolio**
Releated to : 
https://groups.google.com/g/obiba-users/c/hlV4srmHaWM/m/Iy8UAc2aBQAJ

### CODE REVIEW

File: drupal-user-guide/index.rst [hunk 1]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 Version
+  - Got to the Jquery config page under <YourDrupalInstallation>/admin/config/development/jquery_update
+  - In JQueryMigration Section check the  Enable jQuery Migrate Plugin options, (Don't enable the Jquery Migration CDN)
+  - In the Theme Override Section Click on Configure  link of your enabled theme
+  - Once on the your custom theme Configuration page check the 2.2 Theme specific jQuery version under the JQUERY UPDATE Section and click on Save configuration
+  - It may be useful to switch Default jQuery version to 2.2
+ - Clear your site cache
+
+These Steps may Help anyone who update his Drupal installation to 7.72
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Help => help

File: drupal-user-guide/index.rst [hunk 2]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version
+After that please follow these steps:
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 Version
+  - Got to the Jquery config page under <YourDrupalInstallation>/admin/config/development/jquery_update
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Got => Go

File: drupal-user-guide/index.rst [hunk 3]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - In JQueryMigration Section check the  Enable jQuery Migrate Plugin options, (Don't enable the Jquery Migration CDN)
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: JQueryMigration Section => "JQuery Migration" section
 Enable jQuery Migrate Plugin => "Enable JQuery Migrate Plugin"
Don't => do not
JQuery Migration CDN => "JQuery Migration CDN"

File: drupal-user-guide/index.rst [hunk 4]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - In the Theme Override Section Click on Configure  link of your enabled theme
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Theme Override Section Click => "Theme Override" section, click
Configure  => the "Configure"

File: drupal-user-guide/index.rst [hunk 5]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - Once on the your custom theme Configuration page check the 2.2 Theme specific jQuery version under the JQUERY UPDATE Section and click on Save configuration
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Configuration => "Configuration"
Theme specific jQuery => theme specific jQuery
JQUERY UPDATE Section => "JQUERY UPDATE" section
Save => "Save"

File: drupal-user-guide/index.rst [hunk 6]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - It may be useful to switch Default jQuery version to 2.2
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Default jQuery => the default jQuery

File: drupal-user-guide/index.rst [hunk 7]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+ - Clear your site cache
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: indent with one more space

File: drupal-user-guide/index.rst [hunk 8]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
 If some templates have been overridden, please compare with the new original one.
 
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version
```
Comment (ymarcon) [inline drupal-user-guide/index.rst:241]: end your sentence with a .

**Review by ymarcon — CHANGES_REQUESTED** [review]

File: drupal-user-guide/index.rst [hunk 9]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+After that please follow these steps:
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 Version.
+  - Go to the Jquery config page under <YourDrupalInstallation>/admin/config/development/jquery_update.
+  - In  "JQuery Migration" Section check the   "Enable JQuery Migrate Plugin" options, (Do not enable the "JQuery Migration CDN").
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Section => section
Do not => do not

File: drupal-user-guide/index.rst [hunk 10]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - In the "Theme Override" Section Click on the "Configure"  link of your enabled theme.
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Section Click => section, click

File: drupal-user-guide/index.rst [hunk 11]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - Once on the custom theme "Configuration" page check the 2.2  theme specific jQuery version under the  "JQUERY UPDATE" Section and click on "Save" configuration.
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Section => section

File: drupal-user-guide/index.rst [hunk 12]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - It may be useful to switch the default jQuery version to 2.2.
+  - Clear your site cache
+
+These Steps may help anyone who update his Drupal installation to 7.72.
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Steps => steps

File: drupal-user-guide/index.rst [hunk 13]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version
+After that please follow these steps:
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 Version.
+  - Go to the Jquery config page under <YourDrupalInstallation>/admin/config/development/jquery_update.
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Jquery config => jQuery configuration

File: drupal-user-guide/index.rst (hunk 8 above)
Comment (ymarcon) [inline drupal-user-guide/index.rst]: End your sentence with a .

File: drupal-user-guide/index.rst [hunk 14]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
 
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version
+After that please follow these steps:
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 Version.
```
Comment (ymarcon) [inline drupal-user-guide/index.rst]: Version => version

**Review by ymarcon — CHANGES_REQUESTED** [review]

**Review by ymarcon — APPROVED** [review]

File: drupal-user-guide/index.rst [hunk 15]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
 If some templates have been overridden, please compare with the new original one.
 
 If you have defined a sub-theme of obiba_bootstrap's theme, you might need to update your style sheet.
+
+Drupal 7.72
+-----------
+
+For the security issues you should update your Drupal Installation to 7.72 Latest version, make sure to use the bootstrap-7.x-22 version.
```
Comment (kazoompa) [inline drupal-user-guide/index.rst]: Suggestion:

For the security issues you should update your Drupal installation to the latest version 7.72. Also,  make sure to use the bootstrap-7.x-22 version.

File: drupal-user-guide/index.rst [hunk 16]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+After that please follow these steps:
+
+  - Install and enable the jquery_update-7.x-3.0-alpha5 version.
+  - Go to the jQuery configuration page under <YourDrupalInstallation>/admin/config/development/jquery_update.
+  - In  "JQuery Migration" section check the "Enable JQuery Migrate Plugin" options, (do not enable the "JQuery Migration CDN").
```
Comment (kazoompa) [inline drupal-user-guide/index.rst]: extra blank `In  "JQuery`

File: drupal-user-guide/index.rst [hunk 17]
```diff
@@ -234,3 +234,19 @@ The following instructions apply when upgrading from obiba_mica 7.x-1.3 or older
...
+  - In the "Theme Override" section click on the "Configure"  link of your enabled theme.
+  - Once on the custom theme "Configuration" page check the 2.2  theme specific jQuery version under the  "JQUERY UPDATE" section and click on "Save" configuration.
+  - It may be useful to switch the default jQuery version to 2.2.
+  - Clear your site cache.
+
+These steps may help anyone who update his Drupal installation to 7.72.
```
Comment (kazoompa) [inline drupal-user-guide/index.rst]: These steps are recommended for anyone who updates the Drupal installation to 7.72.

**Pull request merged**